Develop

M4Xpy opened the pull request.

M4Xpy committed:
solve format_linter_error() in app/main.py

M4Xpy committed:
Solve all  three functions in app/main.py

YukihiroSM commented on app/main.py:
```diff
@@ -1,13 +1,30 @@
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    # """ formats all errors for a particular file
+    # and adds the path key — path to the file,
+    # and the status key — "failed" if there are errors,
+    # "passed" if there are no errors:
+    # """
```
if you are using docstrings it's not needed to add them as comments using #
```suggestion
    """ formats all errors for a particular file
    and adds the path key — path to the file,
    and the status key — "failed" if there are errors,
    "passed" if there are no errors:
    """
```

YukihiroSM replied:
https://www.programiz.com/python-programming/docstrings

M4Xpy replied:
i know it ,
but  normal docstring dont  permit your  test ,
and i sended report about  it  already.

So , i must delete docstrings at all ?

YukihiroSM reviewed (changes requested):
The solution is good! I like your comments for functios. The only one question is about docstrings :)

M4Xpy committed:
Update main.py

normal docstring dont permit your test ,
and i sended report about it already.

So , i delete docstrings at all

EdAlekseiev reviewed (approved):
Good job
